ema_workbench/analysis/prim.py

```diff
@@ -1046,8 +1046,8 @@ def __init__(
                 x = x.drop(column, axis=1)
                 _logger.info(
                     (
-                        "{} dropped from analysis " "because only a single category"
-                    ).format(column)
+                        f"{column} dropped from analysis " "because only a single category"
+                    )
```

Comment:
I think both the break in stings `" "` could be removed, as well as the extra set of parentheses.